### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #11: Create PR template
Create a PR template that facilitates the linking of issues and PR's so that the issue will be closed automatically when a PR is merged.

### PR DESCRIPTION
**Pull request by JetPlaneJJ**
# Description
Added a markdown file for PR templates. Got some ideas from [appium's PR template](https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

# Type of changes
- [ ] Bugfix / resolves #11.
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+*A clear and concise description of what the pull request is about.*
+
+# Type of changes
+- [ ] Bugfix / resolves *#issue_number*
+- [ ] New Feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
+
+# Screenshots (optional)
+*Insert screenshots of your work.*
```
Comment (NickSchimek) [inline pull_request_template.md:12]: Very nice! 👍

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Description
+*A clear and concise description of what the pull request is about.*
+
+# Type of changes
+- [ ] Bugfix / resolves *#issue_number*
```
Comment (NickSchimek) [inline pull_request_template.md]: Perhaps we can move this `resolves *#issue_number*` to the description section? When I read `Bugfix / resolves *#issue_number*` I get the impression that only bugfixes should be linked to issues when, ideally, all PR's should be linked to an open issue.

**Review by NickSchimek — CHANGES_REQUESTED** [review]
Thanks! I have one suggested change

File: pull_request_template.md (hunk 2 above)
Comment (JetPlaneJJ) [reply pull_request_template.md]: sure!

**Review by NickSchimek — APPROVED** [review]
Awesome! Thanks 😊

**Pull request merged**